src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -19,60 +19,177 @@ config:
     endpoint: 1
 
 tests:
-    ### MANDATORY GLOBAL Attributes
-    ### Attribute[0xFFFD]: ClusterRevision =======================================
-    - label: "2: read the global attribute: ClusterRevision"
+    - label: "Read the global attribute: ClusterRevision"
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
+          value: 5
           constraints:
               type: uint16
-              minValue: 5
-              maxValue: 200
+              minValue: 1
```

Comment:
Matter revision start at 5 for WindowCovering Earlier revision are Zigbee ZCL so there is nothing below
 We can stop at 5 but are you gonna be here to update this value every year each time we bump a version # ?

Reply:
ok we will go back to the min max value check